common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerValidator.java

```diff
@@ -60,11 +61,25 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.Set;
 
 public class BrokerValidator {
 
     private static final String TAG = "BrokerValidator";
 
+    private static boolean shouldTrustDebugBrokers = BuildConfig.DEBUG;
+
+    public static void setShouldTrustDebugBrokers(final boolean shouldTrustDebugBrokers) {
+        if (!BuildConfig.DEBUG && shouldTrustDebugBrokers) {
```

Comment:
shouldTrustDebugBrokers and BuildConfig.DEBUG are identical.  Did you mean to use a different BuildConfig string for that configuration?

Reply:
The syntax highlighting got me.  Could we name the method parameter and the field differently?

Reply:
No they are not identical....`shouldTrustDebugBrokers` could have any value passed in when this method is called and that would be independent of the value of `BuildConfig.DEBUG`

Reply:
Yup done already here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1191/commits/dc80e2672613294a6bc0b3252146938f17f04f36